test_rclcpp/test/test_timer.cpp

```diff
@@ -134,3 +130,11 @@ TEST(CLASSNAME(test_time, RMW_IMPLEMENTATION), timer_during_wait) {
   std::chrono::duration<float> diff = (end - start);
   printf("running for %.3f seconds\n", diff.count());
 }
+
+int main(int argc, char ** argv)
+{
+  // NOTE: use custom main to ensure that rclcpp::init is called only once
+  rclcpp::init(argc, argv);
```

Comment:
Does this still work when passing GTest specific command line arguments? Shouldn't  it continue to pass the arguments `0, nullptr`?